Add instruction to remove comments in member.md (#98)
Women-Devs-SG/womendevssg: src/content/members/member.md

<!-- This is a sample file to guide new members on how to add themselves to the community page. -->

> <!-- After you've edited this file with your details, please DELETE these TWO lines of comments (including the first < and last >), before you commit your changes. -->

Reviewer: Hi @FTSE100, if you agree with the suggestion, you can commit it using the GitHub UI below, or make a new commit to your `FTSE100-remove-mbr-comments` branch and this PR will be automatically updated.

```suggestion
<!-- After copying this file to your_name.md, please DELETE these TWO lines of comments. -->
```
